packages/kotti-ui/source/kotti-comment/types.ts

```diff
@@ -58,15 +58,30 @@ export namespace KottiComment {
 		}
 	}
 
+	export type UserData = Pick<PropsInternal, 'userName' | 'userId'>
+
+	export namespace CommentHeader {
+		export const schema = commentSchema
```

Comment:
why would this be on the schema of the CommentHeader @santiagoballadares 

in any case, this needs to be passed on a comment schema level

so the `isInternalThread` needs to be a prop of the commentSchema

not extended on the schema of the commentHeader alone

same for `isModified`